rdrf/registry/patients/models.py

```diff
@@ -261,6 +261,28 @@ class Meta:
         )
 
     @property
+    def code_field(self):
+        gender_string = ""
+
+        if self.sex == "1":
+            gender_string = "Male"
+        elif self.sex == "2":
+            gender_string = "Female"
+        elif self.sex == "3":
+            gender_string = "Indeterminate"
+        else:
```

Comment:
you could use a dictionary here

gender_map = {"1": "Male",...}

Also we need translation marker  _("Male")